src/marqo/s2_inference/processing/utils.py

```diff
@@ -0,0 +1,834 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
If most of these utils come from an OSS package - maybe rename it? For example "oss_utils.py"